### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Shevtsooov**

### CODE REVIEW

**Review by amashutikov — APPROVED** [review]
Good job, in my opinion its better to create const values for discounts.

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+  let totalAmount = 0;
+
+  if (days < 3) {
```
Comment (silchyn) [inline src/calculateRentalCost.js:36]: what is `3`?

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,24 @@
...
+    totalAmount = rentPerDay * days;
+  }
+
+  if (days >= 3) {
+    totalAmount = (rentPerDay * days) - threeDaysDiscount;
```
Comment (silchyn) [inline src/calculateRentalCost.js:41]: that's hard to understand

**Review by silchyn — CHANGES_REQUESTED** [review]

**Review by serentinos — APPROVED** [review]
Good job!

File: src/calculateRentalCost.js (hunk 1 above)
Comment (silchyn) [reply src/calculateRentalCost.js:41]: still magic number

**Review by silchyn — CHANGES_REQUESTED** [review]
check checklist rule `#3`

**Comment by Shevtsooov** [comment]
@silchyn Thank you for the review, now I got what you meant.
Corrected!

**Review by silchyn — APPROVED** [review]

**Pull request closed without merging**